[2025-02-14T21:39:23.233+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T21:39:23.265+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_run_operator_dag.task1 manual__2025-02-14T21:39:22.078640+00:00 [queued]>
[2025-02-14T21:39:23.281+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_run_operator_dag.task1 manual__2025-02-14T21:39:22.078640+00:00 [queued]>
[2025-02-14T21:39:23.281+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-14T21:39:23.307+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task1> on 2025-02-14 21:39:22.078640+00:00
[2025-02-14T21:39:23.313+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=875) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T21:39:23.314+0000] {standard_task_runner.py:72} INFO - Started process 876 to run task
[2025-02-14T21:39:23.316+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_run_operator_dag', 'task1', 'manual__2025-02-14T21:39:22.078640+00:00', '--job-id', '857', '--raw', '--subdir', 'DAGS_FOLDER/11_run_operator_dag.py', '--cfg-path', '/tmp/tmpd81ek3g2']
[2025-02-14T21:39:23.317+0000] {standard_task_runner.py:105} INFO - Job 857: Subtask task1
[2025-02-14T21:39:23.378+0000] {task_command.py:467} INFO - Running <TaskInstance: call_run_operator_dag.task1 manual__2025-02-14T21:39:22.078640+00:00 [running]> on host ab74d4d90d32
[2025-02-14T21:39:23.616+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_run_operator_dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T21:39:22.078640+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T21:39:22.078640+00:00'
[2025-02-14T21:39:23.617+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T21:39:23.649+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-14T21:39:23.650+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-14T21:39:23.663+0000] {subprocess.py:99} INFO - Output:
[2025-02-14T21:39:28.667+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-14T21:39:28.701+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T21:39:28.702+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=call_run_operator_dag, task_id=task1, run_id=manual__2025-02-14T21:39:22.078640+00:00, execution_date=20250214T213922, start_date=20250214T213923, end_date=20250214T213928
[2025-02-14T21:39:28.756+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T21:39:28.791+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14T21:39:28.795+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
